format/spec.md

```diff
@@ -128,13 +128,13 @@ Tables do not require rename, except for tables that use atomic rename to implem
 
 #### Writer requirements
 
-Some tables in this spec have columns that specify requirements for v1 and v2 tables. These requirements are intended for writers when adding metadata files to a table with the given version.
+Some tables in this spec have columns that specify requirements for v1 and v2 tables. These requirements are intended for writers when adding metadata, manifest files, or manifest lists to a table with the given version.
 
-| Requirement | Write behavior |
-|-------------|----------------|
-| (blank)     | The field should be omitted |
-| _optional_  | The field can be written |
-| _required_  | The field must be written |
+| Requirement | Write behavior                                      |
+|-------------|-----------------------------------------------------|
+| (blank)     | The field is not present in the schema              |
```

Comment:
I like having "Should be omitted" still, even if it isn't present in the schema and that means the same thing.

Reply:
I've reverted this one to the original, I don't think there is a lot of confusion around this one.